### ISSUE TITLE
Refatorar código do plugin de recurso

### ISSUE THREAD

**Issue by alessandrofeitoza** [secultce/plugin-Recourse#9]
### WHY

Precisamos usar o plugin de recurso na versão 7 do Mapas, já existe um plugin feito aqui https://github.com/secultce/plugin-Recourse

### HOW

Antes de iniciar o seu uso, seria interessante aplicar os padrões da PSR-12 e outras boas práticas no código.

### CHECKLIST

- [ ] Refatorar com boas práticas

**Comment by Junior-Shyko** [comment]
Concordo plenamente em uma refatoração, alias, seria bom reconstruir, pq além de usar PSR'S o frontEnd está no AngularJS e sendo assim seria incompatível com a V7.
